### ISSUE TITLE
Problem when granted temporal acces to geolocation in Firefox

### ISSUE THREAD

**Issue by elC0mpa** [w3c/geolocation-api#117]
Hello, I am having some issues when working with geolocation in FireFox, First of all I whill show how this works in Chrome and after that I will show the problem in FireFox.

![image](https://user-images.githubusercontent.com/53531665/151065607-afebc5a7-ff42-4910-8da2-1f9885bc4a23.png)
After location is allowed in this website the button is enabled as shown in the next image.

![image](https://user-images.githubusercontent.com/53531665/151065760-9b26e6d1-1abf-47d9-b829-c63e90b61386.png)
This is who it should works.

Now I will show you the problem. Next image shows initial state in firefox.
![image](https://user-images.githubusercontent.com/53531665/151066544-51e981e3-324a-4b32-9262-e91bcb361951.png)

As shown in the next image, the problem is that when location is allowed temporarily, the API doesn´t detect that user allowed it and as a consequence, the button remains inactive.
![image](https://user-images.githubusercontent.com/53531665/151066956-c8b7475f-cca0-48df-8121-c5209bcfe887.png)

I will show you the code I am using even though there is nothing special on it.
![image](https://user-images.githubusercontent.com/53531665/151067510-196b1df6-8a8a-4d5f-9d6d-c94ac63f9bb9.png)

Basically the point is that when you give location temporarily permissions, the onChange is never executed

Thanks in advance, I hope I could solve this problem.

**Comment by reillyeon** [comment]
This seems like a bug in Firefox, though I think it is also an area where the behavior for temporarily granted permissions could be specified more precisely in the Permissions API. Not firing a `change` event at all is definitely a bug.

Firefox's issue tracker is https://bugzilla.mozilla.org/.

CC @marcoscaceres for visibility into the potential area for clarification in the Permissions API.

**Comment by elC0mpa** [comment]
Thanks @reillyeon, so I need to open a bug in Mozillam is this ok?

**Comment by reillyeon** [comment]
Yes, please file a bug with the Mozilla team.

**Comment by Cyver0** [comment]
Seems not to be fixed yet. I was also not able to find it on [https://bugzilla.mozilla.org/](https://bugzilla.mozilla.org/). But maybe i am just blind.
